packages/syntax-core/src/TextArea/TextArea.tsx

```diff
@@ -51,6 +51,12 @@ type TextAreaProps = {
    * Value of the TextArea
    */
   value: string;
+  /**
+   * Boolean of whether the TextArea should be resizable
+   *
+   * @default true
```

Comment:
@somethiiing should we make the default `none`? I think the current resizing behavior (default "on") breaks in a lot of cases

Reply:
oh, i thought the default should have been none. but i was more concerned about backwards compatability. i can change it